client/analysis.html

```diff
@@ -17,8 +18,88 @@
                 <h1>Youtube Comment Analysis.</h1>
                 <ul>
                    <li>Enter a video name to see what people think!</li>
-                    <li>Click <a href="#">here</a> to learn how I analyze the comments.</li>
+                    <li>Click <a onclick="toggleDisplay()">here</a> to learn more.</li>
```

Comment:
nit: clicking this link shouldn't /toggle/ the lightbox, but show it. It should never hide it (even though I know it would be impossible for a user to click this link while the lightbox is showing). Also `toggleDisplay` is vague and doesn't tell us what display is being toggled. Instead call it `showLightbox` and `hideLightbox` (potentially `toggleLightboxVisibility`)